includes/reports/overview/data-provider/class-sensei-reports-overview-data-provider-students.php

```diff
@@ -117,11 +117,7 @@ public function only_course_enrolled_users( WP_User_Query $query ) {
 	public function add_orderby_custom_field_to_user_query( WP_User_Query $query ) {
 		global $wpdb;
```

Comment:
This line could be cleaned up as well? 🙂

Reply:
Good catch! Removed here: https://github.com/Automattic/sensei/pull/5328/commits/40bdd8406d25d1b8219ac62904ba1bdb8dbe2464